client/app/bundles/course/assessment/components/MaterialUploader/index.jsx

```diff
@@ -12,6 +12,9 @@ const propTypes = {
   enableMaterialsAction: PropTypes.bool.isRequired,
 };
 
+/**
+ * @deprecated Superseded by `FileManager`.
```

Comment:
@purfectliterature you might have miss this and the other 6 comments below.

Reply:
>Should we remove this file then?

Sure. I was keeping the original `MaterialUploader` in case we needed to refer or revert to it. Since `FileManager` is fine now, I will just remove this component.

>you might have miss this and the other 6 comments below.

Thanks for letting me know! These messages were not shown on my side even after unhiding the conversations. I had to empty my Chrome cache and site data before I can see them—weird.